rosbag2_storage/include/rosbag2_storage/storage_interfaces/read_only_interface.hpp

```diff
@@ -45,6 +46,21 @@ class ROSBAG2_STORAGE_PUBLIC ReadOnlyInterface
   virtual void set_filter(const StorageFilter & storage_filter) = 0;
 
   virtual void reset_filter() = 0;
+
+  /**
+   * Get the read only storage package name
```

Comment:
A general rule "has this comment added any more clarity than the name of the function already was doing?" - in this case I would argue that it has not. Maybe language like "Provided to help class loaders find plugins implementing this interface." would make it a yes.

Apply to each of these interface functions.